sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -463,6 +459,18 @@ object Radar {
 
         this.initialized = true
 
+        sendLogConversionRequest(
+            "app_open",
+            callback = object : RadarLogConversionCallback {
+                override fun onComplete(
+                    status: RadarStatus,
+                    event: RadarEvent?
+                ) {
+                    this@Radar.logger.i("Conversion type = ${event?.customType}: status = $status; event = $event")
```

Comment:
from the response payload, it should be `event?.type`
![image](https://user-images.githubusercontent.com/61531269/222757724-8848f1b7-42bd-4bfd-8638-893eac1e3574.png)

Reply:
also from `RadarEvent.kt`

Reply:
from the logConversion signature `fun logConversion(name, metadata, callback)`, should we change `name` to type? since we pass the `RadarEvent` to the client, which doesn't have a `name` field